src/main/java/se/digg/dgc/service/impl/DefaultDGCEncoder.java

```diff
@@ -56,6 +56,11 @@ public String encode(final DigitalGreenCertificate dgc, final Instant expiration
     
     dgc.setV(DGCSchemaVersion.DGC_SCHEMA_VERSION);
     
+    if(dgc.getSub().getFnt() == null && dgc.getSub().getFn() != null)
```

Comment:
I don't know whether I think that the transliteration belongs here. We let the caller be responsible of everything else that is set to the DGC.

Reply:
I understand - however, the output of the transliteration of the name is given by eg. the given name, hence there is no reason for the caller to need to do something about it.

Reply:
By embedding it, the transliteration becomes 'normative'

Reply:
Well, the caller created the `DigitalGreenCertificate` in the first place, so I guess that he should now what he puts in there and whether the name needs transliteration.

Reply:
> I let this one rest for a while.
> 
> I think that the library may offer help classes for transliteration (MrzParser), but it is the caller that decides how the supplied DigitalGreenCertificate object should look like.
> 
> As I understood the discussions in the eHealth working group transliteration is mainly for those countries that do not use a latin alphabet.

I have an idea. We'll make your code configurable. If the encoder is configured to perform transliteration the code runs, otherwise it leaves the DGC as supplied. I'll make an edit.